zkvm/src/contract.rs

```diff
@@ -118,6 +119,11 @@ impl ContractID {
         &self.0
     }
 
+    /// Returns the contractID as a UTXO
+    pub fn as_utxo(&self) -> UTXO {
+        UTXO(self.0)
```

Comment:
In ZkVM, utxo and contractID are synonyms. That doesn't mean we should have two distinct types! Can you replace uses of the UTXO type with the ContractID type?

Reply:
Makes sense! That's even simpler :)